src/DETHRACE/common/car.c

```diff
@@ -48,7 +48,7 @@ void (*ControlCar[6])(tCar_spec*, br_scalar) = {
     &ControlCar3,
     &ControlCar4,
     &ControlCar5,
-    NULL
+    NULL,
```

Comment:
What is our convention about last comma in arrays?
Ancient compilers (gcc 3.x-era) do not support this.
My 2c is to don't care at the moment and do something about this when it's needed.

Reply:
Agree - I don't really mind either way. I don't generally add it myself.